modules/mobile/app/serializers/mobile/v0/triage_team_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class TriageTeamSerializer < ActiveModel::Serializer
```

Comment:
Could this use the fast fast_jsonapi/jsonapi-serializer? Our serializers use the former but we're moving the latter soon. Fast JSON API was developed at Netflix and is faster than ActiveModel serializer (especially at large resource counts), it was recently forked to [jsonapi-serializer](https://github.com/jsonapi-serializer/jsonapi-serializer).

Reply:
I guess it depends how well it plays with the OG secure messaging client and its metadata req, it would be nice to be consistent, but we could also refactor later if we want to get this in for testing.

Reply:
I ran across that when debugging what was going on with the metadata links...I'm in favor of refactoring later.